vagrant/release/devstack/Vagrantfile.dogwood

```diff
@@ -0,0 +1,181 @@
+Vagrant.require_version ">= 1.6.5"
+unless Vagrant.has_plugin?("vagrant-vbguest")
+  raise "Please install the vagrant-vbguest plugin by running `vagrant plugin install vagrant-vbguest`"
+end
+
+VAGRANTFILE_API_VERSION = "2"
+
+MEMORY = 4096
+CPU_COUNT = 2
+
+$script = <<SCRIPT
+if [ ! -d /edx/app/edx_ansible ]; then
+    echo "Error: Base box is missing provisioning scripts." 1>&2
+    exit 1
+fi
+OPENEDX_RELEASE=$1
+export PYTHONUNBUFFERED=1
+source /edx/app/edx_ansible/venvs/edx_ansible/bin/activate
+cd /edx/app/edx_ansible/edx_ansible/playbooks
+
+# Did we specify an openedx release?
+if [ -n "$OPENEDX_RELEASE" ]; then
+  EXTRA_VARS="-e edx_platform_version=$OPENEDX_RELEASE \
+    -e certs_version=$OPENEDX_RELEASE \
+    -e forum_version=$OPENEDX_RELEASE \
+    -e xqueue_version=$OPENEDX_RELEASE \
+    -e NOTIFIER_VERSION=$OPENEDX_RELEASE \
+    -e ECOMMERCE_VERSION=$OPENEDX_RELEASE \
+    -e PROGRAMS_VERSION=$OPENEDX_RELEASE \
+  "
+  CONFIG_VER=$OPENEDX_RELEASE
+  # Need to ensure that the configuration repo is updated
+  # The vagrant-devstack.yml playbook will also do this, but only
+  # after loading the playbooks into memory.  If these are out of date,
+  # this can cause problems (e.g. looking for templates that no longer exist).
+  /edx/bin/update configuration $CONFIG_VER
+else
+  CONFIG_VER="master"
+fi
+
+APPSEMBLER_VARS="-e edx_platform_repo="https://github.com/appsembler/edx-platform.git" \
+  -e edx_platform_version="appsembler/release" \
```

Comment:
If this is the Dogwood vargrantfile, then I think we should target our Dogwood Appsembler branch (`appsembler/feature/nergeDogwood.3` instead of `appsembler/release`